### PR TITLE
docs: add programmatic-errors to README.md

### PR DESCRIPTION
**Pull request by GuyMoses**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,6 +176,28 @@ In case your execution tags on different spans appear on different invocations t
 * Each execution tag key can be at most 50 characters long; the `lumigo.execution_tags.` prefix does _not_ count against the 50 characters limit.
 * Each execution tag value can be at most 70 characters long.
 
+
+### Programmatic Errors
+
+[Programmatic Errors](https://docs.lumigo.io/docs/programmatic-errors) allow you to customize errors, monitor and troubleshoot issues that should not necessarily interfere with the service.
+For example, an application tries to remove a user who doesn't exist. These custom errors can be handled by adding just a few lines of additional code to your application.
```
Comment (kenfinnigan) [inline README.md:183]: The word "handled" for me has implications that the issue is solved. Maybe "captured" describes what is happening more accurately?

File: README.md [hunk 2]
```diff
@@ -176,6 +176,28 @@ In case your execution tags on different spans appear on different invocations t
...
+
+Programmatic Errors are labeled as warnings indicating that a non-fatal error occurred, such as an application error. You can log programmatic errors, track custom error issues, and trigger [Alerts](https://docs.lumigo.io/docs/event-alert).
```
Comment (kenfinnigan) [inline README.md:185]: > Programmatic Errors are labeled as warnings indicating that a non-fatal error occurred, such as an application error.

This phrasing is confusing to me, as it says they're warnings but then refers to them as application errors.

File: README.md [hunk 3]
```diff
@@ -176,6 +176,28 @@ In case your execution tags on different spans appear on different invocations t
...
+
+
+#### Creating a Programmatic Error
+
+Programmatic errors are created by adding [span events](https://opentelemetry.io/docs/instrumentation/python/manual/#adding-events) by a custom event attribute with the key name `lumigo.type`.
```
Comment (kenfinnigan) [inline README.md:190]: I'd suggest changing "by a custom event attribute" to "with a custom attribute", to make it clear it's a span event with a custom attribute being set

**Review by kenfinnigan — COMMENTED** [review]
I have a few minor suggestions

File: README.md (hunk 1 above)
Comment (GuyMoses) [reply README.md:183]: yes!

File: README.md (hunk 3 above)
Comment (GuyMoses) [reply README.md:190]: excellent

**Review by kenfinnigan — APPROVED** [review]

**Pull request merged**